helm/idol-nifi/values.yaml

```
@@ -97,21 +97,26 @@ nifi:
     minReplicas: 1
     # -- the maximum size of the nifi statefulset
     maxReplicas: 8
+    # -- no. of seconds a limit must be exceed before scaling the nifi statefulset
+    stabilizationWindowSeconds: 300
     # -- one or more metrics controlling the horizontal pod autoscaler (https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2/#HorizontalPodAutoscalerSpec)
     metrics:
```

```suggestion
    # -- one or more metrics controlling the horizontal pod autoscaler (https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2/#HorizontalPodAutoscalerSpec)
    # @default -- See below
    metrics:
```
Slightly nicer for README generation